src/Sarif.ValidationTests/Sarif.ValidationTests.csproj

```diff
@@ -31,25 +31,13 @@
   </ItemGroup>
 
   <ItemGroup>
-    <None Include="$(SolutionDir)\Sarif\Schemata\Sarif.schema.json">
-      <CopyToOutputDirectory>PreserveNewest</CopyToOutputDirectory>
-    </None>
+    <None Include="$(SolutionDir)\Sarif\Schemata\Sarif.schema.json"
+          CopyToOutputDirectory="PreserveNewest" />
+    <None Include="$(SolutionDir)\Sarif.FunctionalTests\v1\**\*.sarif"
+          CopyToOutputDirectory="PreserveNewest"
+          LinkBase="v1" />
```

Comment:
`LinkBase` is a new feature added in .NET SDK 2.0 -- and a good thing, too, because without it, you end up with
```
OutputPath\
  ConverterTestFiles\
  ...
```
rather than what you want, which is
```
OutputPath\
  v1\
    ConverterTestFiles\
    ...
```
See Microsoft/msbuild#2949.